### PR TITLE
[9.5] update changelog for point release

### PR DESCRIPTION
**Pull request by tamiko**

### CODE REVIEW

File: doc/news/9.5.0-vs-9.5.2.h [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  Fixed: various configuration, compilation and instantiation issues have
+  been addressed:
+  <ol>
+    <li>fix a compilation issue with bundled boost and modern clang compiler</li>
+    <li>instantiate various MPI helper functions for signed long long int</li>
+    <li>add missing codimension-one instantiation for some DoFTools functions</li>
+    <li>Fix compiling with PETSc with complex scalar type</li>
+    <li>Allow compilation with PETSc but without MPI</li>
+    <li>fix compilation of bundled tbb with gcc-13</li>
+    <li>detect Trilinos NOX support and guard Trilinos NOX usage</li>
```
Comment (bangerth) [inline doc/news/9.5.0-vs-9.5.2.h:41]: Please make the upper/lowercase spelling of the first letter of each entry consistent.

File: doc/news/9.5.0-vs-9.5.2.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  </ol>
+  <br />
+  (Daniel Arndt, Wolfgang Bangerth, Matthias Maier, 2023/01/31)
```
Comment (bangerth) [inline doc/news/9.5.0-vs-9.5.2.h:44]: Why only the three of us?

File: doc/news/9.5.0-vs-9.5.2.h (hunk 1 above)
Comment (tamiko) [reply doc/news/9.5.0-vs-9.5.2.h]: Done.

File: doc/news/9.5.0-vs-9.5.2.h (hunk 2 above)
Comment (tamiko) [reply doc/news/9.5.0-vs-9.5.2.h:44]: @bangerth The three of us are the authors of the commits that have been added on top of 9.5.0.

**Review by masterleinad — APPROVED** [review]

File: doc/news/9.5.0-vs-9.5.2.h (hunk 2 above)
Comment (bangerth) [reply doc/news/9.5.0-vs-9.5.2.h]: I see. I personally don't see that as particularly relevant given that others have contributed in other ways. But it's probably also not important given that the release is going to be published by us all.

**Review by bangerth — APPROVED** [review]

File: doc/news/9.5.0-vs-9.5.2.h (hunk 2 above)
Comment (tamiko) [reply doc/news/9.5.0-vs-9.5.2.h]: @bangerth I am sorry, I am genuinely confused.

I just wanted to have a small changelog entry showing up here [1] similarly to what we - admittedly not very consistently - did in the past, exemplarily [2,3].

Let us have a discussion about this offline.

[1] https://dealii.org/current/doxygen/deal.II/pages.html
[2] https://dealii.org/current/doxygen/deal.II/changes_between_9_3_0_and_9_3_1.html
[3] https://dealii.org/current/doxygen/deal.II/changes_between_9_3_1_and_9_3_2.html

**Pull request merged**